src/vector/_backends/awkward_.py

```diff
@@ -275,7 +269,7 @@ def _wrap_result(
         Wraps the raw result of a compute function as an array of scalars or an
         array of vectors.
         """
-        if returns == [float] or returns == [bool]:
+        if returns in [[float], [bool]]:
```

Comment:
With a short list like this, I thought the use of `==` (the most common operator for `if`) would be more clear than `in`. There's only two possibilities.

Reply:
This expands much better if you ever add an option, and is less repetitive even for two. But especially for lists of lists (you can't make it a set, since lists can't go in a set), it's not necessary that much better.